wiki/framework/settings-framework.md

```diff
@@ -103,3 +105,40 @@ class ACE_Settings {
 As stated before, the server config gets loaded through the optional `ace_server.pbo`. This PBO is only required (and should only be used) on the server - clients do not need to have this! It is for this reason we have not signed this PBO.
 
 Load the `ace_server.pbo` like any other addon on your server. It is advised to create an `@aceServer` mod folder with an `addons` sub folder where you would paste the `ace_server.pbo` and load that through `-serverMod=@aceServer`.
+
+
+### 3.3 Changing ACE_settings dynamically with Mission Parameters
+
+It is possible to change ace settings via a mission's params.
```

Comment:
`ACE3 settings` via [Mission Parameters](https://community.bistudio.com/wiki/Arma_3_Mission_Parameters)
maybe add a link to: https://community.bistudio.com/wiki/Arma_3_Mission_Parameters